grammars/gfm.cson

```diff
@@ -774,4 +774,30 @@
     'end': '--\\s*>'
     'name': 'comment.block.gfm'
   }
+  {
+    'begin': '(^\\|)'
+    'end': '(\\|$)'
+    'captures':
+      '1':
+        'name': 'border.pipe.outer'
+      '2':
+        'name': 'border.pipe.outer'
+    'name': 'table.gfm'
+    'patterns': [
+      {
+        'match': '(:?)(-+)(:?)'
+        'captures':
+          '1':
+            'name': 'border.modifier'
```

Comment:
Would `border.alignment` make more sense as the scope name here since the `:` always refers to the column's alignment right? Or maybe `justify` instead? Then it could be `table.border.justify.left`?